azure-pipelines.yml

```diff
@@ -0,0 +1,41 @@
+# CppRestSdk Azure Pipelines Configuration
+
+jobs:
+  - job: Ubuntu_1604_Apt_Debug
+    pool:
+      vmImage: 'Ubuntu 16.04'
+    steps:
+    - script: |
+        sudo apt-get install -y ppa-purge
+        sudo ppa-purge -y ppa:ondrej/php
```

Comment:
@alexkaratarakis Thought you might find that amusing :P

Thanks for the help again!